_episodes/13-looking-up-data.md

```diff
@@ -49,7 +49,7 @@ The next exercise demonstrates this two stage process in full.
 >* Give the column a name e.g. "Journal details"
 >* In the expression box you need to write some GREL where the output of the expression is a URL which can be used to retrieve data (the format of the data could be HTML, XML, JSON, or some other text format)
 >
->In this case we are going to use the CrossRef api (read more about the CrossRef service at [http://www.crossref.org](http://www.crossref.org), read more about the API we are going to use at [https://github.com/CrossRef/rest-api-doc/blob/master/rest_api.md)](https://github.com/CrossRef/rest-api-doc/blob/master/rest_api.md))
+>In this case we are going to use the CrossRef api (read more about the CrossRef service at [http://www.crossref.org](http://www.crossref.org), read more about the API we are going to use at [https://github.com/CrossRef/rest-api-doc)](https://github.com/CrossRef/rest-api-doc))
```

Comment:
Has this got an extra `)` at the end of the URL?
I know you were just updating the URL, but wonder if it might improve the readability if the para could be re-written to avoid the use of brackets in the text?

Reply:
Just updated it. How does it look now?